dev/Dockerfile

```diff
@@ -1,18 +1,4 @@
-ARG IMAGE_NAME
-FROM ${IMAGE_NAME}
+# FROM astrocrpublic.azurecr.io/runtime:3.0-4
```

Comment:
Did we try to run the provider against Airflow 3? If yes, it may be worth to update https://github.com/astronomer/airflow-provider-fivetran-async/issues/110 - otherwise, it may be worth to add this line only when that work is done.